tasks/Restaurant.tex

```diff
@@ -0,0 +1,90 @@
+          \item Needing assistance for picking up an item/the tray causes a reduction of 10\% in scoring.
+          \item Have a human place the item in a basket causes a reduction of 20\% in scoring.
+          \item Have the quest take the item out of the basket causes a reduction of 20\% in scoring.
+          \item Asking the barman at which side the \textit{Kitchen-bar} is causes a reduction of 5\% in the total scoring.
+        \end{itemize}
+
+	\item \textbf{Start:} There is no fixed start signal in this test, it starts when both robots are ready (though within a reasonable time).
+
+	\item \textbf{Order:} The way the user provides information to the robot is up to the robot's team. A natural interaction is preferred.
+
+	\item \textbf{Location:} This test can be arranged in any real restaurant or shopping mall. If this is not possible, the test can be conducted in an arbitrary room containing the appropriate locations.
+	  The only requirement is that this room is not part of the arena and that the teams do not know the room beforehand.
+	  The exact location, including the object and delivery locations, will be defined by the technical committee on site (and in corporation with the local organization).
+	  In addition, to avoid unnecessary time investment for navigation, the distances between tables and the \enquote{Kitchen Bar} will be minimal.
+
+	\item \textbf{Disturbances from outside:} If a person from the audience (severely) interferes with the robot in a way that makes it impossible to solve the task, the teams may repeat the test immediately.
+
+	\item \textbf{Instruction:} The robot interacts with the operators, not the team. That is, the team is not allowed to instruct the \textit{Professional Barman}. All instructions should be provided by the robot itself.
+	It is not allowed to the team to instruct the clients on how to get robot's attention. It shall be done in a natural way like when interacting with a human waiter.
+
+    \item \textbf{Map:} The area of the test is not allowed to be mapped before the test. Mapping can be started just before starting the robot.
+
+    \item \textbf{Two robots:} This test is executed with two robots working parallel. A robot driving into an other robot is disqualified from this test. A robot preventing the other robot to execute this test by (severely) interferes is also disqualified from this test.
+
+\end{enumerate}
+
+\subsection{Referee instructions}
+
+The referee needs to
+\begin{itemize}
+	\item Prepare orders for each client in advance, so that there can be no confusion. These orders must also be available at the kitchen.
+\end{itemize}
+
+\subsection{OC instructions}
+During Setup days:
+\begin{itemize}[nosep]
+	\item Check with local (security) management if the possible location can be used for the restaurant test.
+\end{itemize}
+
+1 hour before the test:
+\begin{itemize}[nosep]
+	\item Gather all teams and robots to move to the location of the test.
```

Comment:
move teams to some nearby queuing area and instruct the teams how/when to move to the actual restaurant